en/app.ftl

```diff
@@ -127,6 +127,14 @@ whatsnew-feature-alias-to-mask-snippet = Notice a change? We’re now calling al
 # Thus, if your locale did not make this change, please keep the English version.
 whatsnew-feature-alias-to-mask-description = Notice a change? We’re now calling aliases “masks” to make { -brand-name-firefox-relay } easier to use and open the door for new features.
 
+whatsnew-feature-premium-expansion-sweden-heading = { -brand-name-relay-premium } available in Sweden
+whatsnew-feature-premium-expansion-finland-heading = { -brand-name-relay-premium } available in Finland
+# A preview of the full content of `whatsnew-feature-premium-sweden-description`.
+# When translating, please make sure the resulting string is of roughly similar
```

Comment:
This is unlikely to happen, and we shouldn't ask locales to do it…

With that said: does this apply to the first string, or both?

Reply:
I don't know of a good way to do this programmatically, since string length doesn't necessarily correspond to the number of characters, and I think there isn't even a definition of character that represents e.g. Latin letters but doesn't risk splitting glyphs spanning multiple Unicode code points (or something like that - I don't know too much of it) into two.

So it applies to the first string: `-snippet` is an excerpt of `-description`.

Reply:
Oh, I see now. That's something only Relay does, and I would strongly suggest to stop doing that. 

It's based on the assumption that other locales can create a meaningful sentence by cropping the full description at the start :-( 

It would make more sense to crop the translation via CSS, or even better, always design a system that allows to display the full sentence.

Reply:
I'll bring it up with UX to see if they can come up with a design that avoids the use of snippets.

Unfortunately, I don't think CSS has the ability to truncate something at the two-line cut-off. That said, I don't think this needs toe be a meaningful sentence (not all English strings are either)? It's essentially a teaser showing that you can click through to get the full message, e.g. from this:

![image](https://user-images.githubusercontent.com/4251/167644021-a266aae3-5fd1-46f6-933b-40e5ea639f5a.png)

to this:

![image](https://user-images.githubusercontent.com/4251/167644074-fb055f2b-8bc1-4c64-8826-8ce3fa704f31.png)